Banyan/src/pfs.jl

```diff
@@ -47,6 +47,7 @@ function ReadBlock(
     path = getpath(loc_params["path"])
     if (loc_name == "Remote" && (occursin(".h5", loc_params["path"]) || occursin(".hdf5", loc_params["path"]))) ||
         (loc_name == "Disk" && HDF5.ishdf5(path))
+        # @show isfile(path)
```

Comment:
Should this always get printed? or only in debug mode

Reply:
No